### PR TITLE
Add name

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,6 @@ After adding the name execute update.py to update the readme.
 * Avinash Agarwal [manu4rhyme](https://github.com/manu4rhyme)
 * Dushyant Mishra [zero1hac](https://github.com/zero1hac)
 * Govind Sahai [mafiya69](https://github.com/mafiya69)
-* Harshit [iamharshit] (https://github.com/iamharshit)
```
Comment (leosartaj) [inline README.md:29]: Why was this removed?

**Review by leosartaj — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Vaibhavdixit02) [reply README.md:29]: Must have been by mistake. Did not pay attention . Sorry.

Comment (leosartaj) [reply README.md:29]: No, I guess his name was never added to AUTHORS.md. If you add there and run the script, it should be good.

Comment (Vaibhavdixit02) [reply README.md:29]: I did sir, but there seems to be some error and it is saying that the Travis CI build failed.

**Review by leosartaj — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ After adding the name execute update.py to update the readme.
...
+* Harshit  [iamharshit](https://github.com/iamharshit)
```
Comment (leosartaj) [inline README.md:29]: Don't add manually run the script.

Comment (leosartaj) [reply README.md:29]: Just replace `Harshit  [iamharshit](https://github.com/iamharshit)` with 
`Harshit [iamharshit](https://github.com/iamharshit)`

Comment (Vaibhavdixit02) [reply README.md:29]: Done.

**Comment by leosartaj** [comment]
Thanks! It's in.

**Pull request merged**